PostDeploymentActions/updateMasterTimer.ps1

```diff
@@ -0,0 +1,8 @@
+$date = Get-Date
+$min = $date.Minute % 15 + 1
```

Comment:
@kkuzmin - my answer is showed as outdated as you have changed  the line.. 

I am against keeping +1 as it will run only 3x per hour if `$min` is 15. It is even clearly showed in the link you mentioned.

Reply:
The goal of this change is to adjust schedule to trigger Master function as soon as possible. Taking into the account that post deployment actions take less than 1 min (just two file writes). If we remove +1 then first Master function call will be delayed for 15 mins.
DO you have any better suggestions how to achieve this?
BTW, if Master will be called 3 times only if `Get-Date` return 14:00 (min:sec), so chances are very low and it is not critical.

Reply:
I thought you already suggested the solution. You can add 59s there which gives us this "minute" which should be enough if we believe that deployment actions take less then 1min. 
I think it is much more crucial that every function is triggered 4x per hour with small a chance that we may miss initial call after deployment than having some functions which run 3x and some 4x..
BTW it is modulo so the chance is exactly 4  .